### ISSUE TITLE
MSI Mystic Lights?

### ISSUE THREAD

**Issue by RedSoxFan04** [nagisa/msi-rgb#21]
Does msi-rgb work with MSI's "Mystic Lights" or does it only control LED strips which are connected to the RGB header?

**Comment by nagisa** [comment]
Unclear. On my motherboard it controls the 5050 header and the LEDs on the motherboard itself, but control happens through a independent chip, which may also implement support for mystic lights or it might not.

MSI being a :hankey: company it is, you really cannot tell without having hardware in hand.

**Comment by RedSoxFan04** [comment]
I might be confused. Is Mystic Light the lights themselves or is it just the name of MSI's special program for controlling the lights?

**Comment by dcerisano** [comment]
Mystic Light is their hankey proprietary sync program for all

**Comment by RedSoxFan04** [comment]
Is JLED1 a header on the motherboard which is intended for extra light strips which can be added inside the case?  So does msi-rgb just get the multicolored lights on the board working, like on the (Northbridge?) heatsink on some of MSI's boards?

As to the issue of needing separate programs for lighting, I guess I could make shell scripts for making all of my RGB devices the same color or for different color patterns. I think OpenRazer is accessible via the comand-line, so that should be possible for me.

**Comment by dcerisano** [comment]
Yes. You can see that RGB technology is extremely fragmented, requiring
custom integrations for every new controller chip.
Msi-rgb is just one example for one SIO chip (Nuvoton).

*Dominic Cerisano*
Director Mobile VR Labs
www.standard3d.com <https://standard3d.com>
dcerisano@standard3d.com
ca.linkedin.com/in/dcerisano
dcerisano@skype

**Comment by ayufan** [comment]
Take a look at: https://github.com/nagisa/msi-rgb/issues/54#issuecomment-463798374
